pkg/ovsdb/handler.go

```diff
@@ -44,7 +44,7 @@ func (ch *Handler) Transact(ctx context.Context, param []interface{}) (interface
 	txn := NewTransaction(ch.etcdClient, req)
 	txn.schemas = ch.db.GetSchemas()
 	txn.Commit()
-	klog.V(5).Infof("Transact response %v", txn.response)
+	klog.V(5).Infof("Transact response %s", &txn.response)
```

Comment:
why pointer? it should work also without ampercent

Reply:
The String method defined for `*TransactResponse`. Without ampercent it doesn't work, or requires explicitly call String()